File: run_git.js
const { execSync } = require('child_process');
const fs = require('fs');

try {
  let out = "";
  console.log("Adding...");
  out += execSync('git add .').toString() + "\\n";
  console.log("Committing...");
  try {
    out += execSync('git commit -m "Refactor: Finalized Hackathon Release (TalentScout AI)"').toString() + "\\n";
  } catch (e) {
    out += "Commit output (might be empty): " + e.stdout + "\\n";
  }
  console.log("Pushing...");
  out += execSync('git push origin HEAD --force').toString();
  fs.writeFileSync('git_output.txt', out);
  console.log("Done.");
} catch (error) {
  fs.writeFileSync('git_output.txt', "ERROR: " + error.message + "\\n" + (error.stdout ? error.stdout.toString() : ""));
}

File: run_git.py
import subprocess

commands = [
    ["git", "add", "."],
    ["git", "commit", "-m", "Refactor: Cleanup repo and rename to TalentScout AI for Hackathon"],
    ["git", "push", "origin", "main"]
]

with open("git_output.txt", "w") as f:
    for cmd in commands:
        f.write(f"\\n=== Running {' '.join(cmd)} ===\\n")
        try:
            result = subprocess.run(cmd, cwd=r"c:\\Users\\dell\\Desktop\\xnords", capture_output=True, text=True)
            f.write(result.stdout)
            if result.stderr:
                f.write("\\n[ERROR]\\n" + result.stderr)
        except Exception as e:
            f.write(f"\\nException: {e}\\n")
